pkg/admin/data/data_manager.go

```diff
@@ -235,10 +244,25 @@ func copyPath(src, dest string) error {
 		klog.InfoS("Failed to copy", "cmd", cmd,
 			"stdout", strings.ReplaceAll(outb.String(), "\n", `, `),
 			"stderr", errb.String())
-		return fmt.Errorf("failed to copy %q to %q: %w", src, dest, err)
+		copyErr := fmt.Errorf("failed to copy %q to %q: %w", src, dest, err)
+		if err := os.RemoveAll(tmpDest); err != nil {
```

Comment:
We should probably add "removing intermediate path" message similar to below.

Reply:
I added the information to the error message, I don't think we need extra log for this